types/chance/chance-tests.ts

```diff
@@ -18,7 +18,7 @@ const strArr: string[] = chance.n(chance.string, 42);
 const strArr2: string[] = chance.n((a) => a.value, 42, { value: 'test' });
 
 const uniqInts: number[] = chance.unique(chance.integer, 99);
-const uniqInts2: number[] = chance.unique(a => a.value, 99, { value: 1 });
+const uniqInts2: number[] = chance.unique(a => chance.integer({ min: 0, max: 999 }) + a.value, 99, { value: 1000 });
```

Comment:
I'm not going to remove the the uniqInts2 case as it's testing weather or not the passed in values come back through the options. But yeah that should be included.

Reply:
Also, problem with this. Unless I'm mistaken, the interfaces and types in this library are not importable. I created a dummy interface for the test file, but I don't like it. Would there be an issue exposing these types?

Reply:
ah, good point. it's actually already exported as `Chance.Currency`, not `Currency` like my example indicates. so i think would actually be 

```typescript
const currencyComparator = (arr: Chance.Currency[], value: Chance.Currency): boolean => arr.findIndex(x => x.code === value.code && x.name === value.name) > -1;
```

you should be able to replace your `currencyType` interface with that.

Reply:
I've already tried :/ 
![image](https://user-images.githubusercontent.com/12650208/71124319-71bac580-2199-11ea-9f8f-9876814cc874.png)

Reply:
Ah, i see the problem. it's in the namespace, but not exprted. i misread it on my phone.

frankly, i think things like `Currency` _should_ be exported. if you look at the usage of `currency_pair` in the following test, you see that the object that's returned is _not_ typed. because the return types aren't exported, i don't think a user of the lib can appropriately type things that are returned. most of our tests are returning basic types so that's probably why it hasnt mattered so far.

for now, LGTM. this exporting thing ill try to take a look at when i have time.

Reply:
My thoughts exactly. I appreciate you looking this over!